java/bogdana_maksiuta/src/com/zoolatech/lecture2/tasks/_2/Robot.java

```diff
@@ -0,0 +1,189 @@
+package com.zoolatech.lecture2.tasks._2;
+
+/**
+ * Please carefully read the task description: you need to have a class that provides following methods:
+ *
+ * move()
+ * move(int tiles)
+ * turnLeft()
+ * turnRight()
+ * currentLocation()
+ * currentDirection()
+ * constructors to properly create an object of this class
+ */
+public class Robot {
+    private int h = 0;
```

Comment:
`0` is the default value for int variables, so, usually, such assignments can be omitted